### DISCUSSION TITLE
CORS issues when deployed with domain name

### DISCUSSION THREAD

**Discussion by DenisIvanovIvanov** [jasontaylordev/CleanArchitecture#572] [General]
I'm using the template to create small web app for my wife. 

I've finally deployed it with domain name and SSL, but login redirect doesn't work because of CORS. The error is
`CORS request made for path: /sockjs-node/701/xvyuio0h/xhr from origin: https://domainname.com but was ignored because path was not for an allowed IdentityServer CORS endpoint"`

I've tried enabling all cors as suggested
```
services.AddSingleton<ICorsPolicyService>((container) => {
     var logger = container.GetRequiredService<ILogger<DefaultCorsPolicyService>>();
     return new DefaultCorsPolicyService(logger)
     {
          AllowAll = true
      };
});

services.AddCors(options =>
{
     options.AddPolicy(name: "_myAllowSpecificOrigins",
               builder =>
               {
                    builder.AllowAnyHeader()
                                  .AllowAnyMethod()
                                  .AllowAnyOrigin();
               });
      });
```

And using it
```
app.UseRouting();
app.UseCors("_myAllowSpecificOrigins");
```

but it still doesn't work. Can someone help on that?

**Comment by ITEAlaaEddin** [comment]
which setting do u change in angular to use your certificate ?
